examples/from_settings.rs

```diff
@@ -0,0 +1,108 @@
+//! Creates a context from a settings struct.
```

Comment:
I think this should be just a doc example for `XyAxes`. No the whole main with systems, just the settings struct and preset creation from it to showcase it.

Reply:
Added a doc example to `Cardinal` and deleted this file